packages/backend/src/controllers/house/dto/house-tasks-response.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { TaskDocument } from '../../../db/task/task.schema';
+
+export default class HouseTasksResponseDto {
+  @ApiPropertyOptional()
+  tasks: TaskDocument[];
```

Comment:
can we have a task response dto? I'm just thinking that the completed property should be presented as a boolean

Reply:
so include all the fields but last_completed is instead isComplete with true or false

Reply:
yeah exactly, or just `completed` is fine